pages/en/blog/announcements/v22-release-announce.md

```diff
@@ -0,0 +1,96 @@
+---
+date: '2024-04-17T15:45:00.000Z'
+category: announcements
+title: 'Node.js 22 is now available!'
+layout: blog-post
+author: The Node.js Project
+---
+
+We're excited to announce the release of Node.js 22!
+Highlights include require()ing ESM graphs, WebSocket client, updates of the V8 JavaScript engine, and more!
+
+As a reminder, Node.js 22 will enter long-term support (LTS) in October, but until then, it will be the "Current" release for the next six months.
+We encourage you to explore the new features and benefits offered by this latest release and evaluate their potential impact on your applications.
+
+The project continues to make progress across a number of areas, with many new features and fixes flowing into existing LTS releases.
+For that reason, the changes outlined in the [changelog][CHANGELOG] for Node.js 22 only represent a small subset of the features and
+work since the last major release. This blog post will add some additional context on the larger body of work in relation to those changes.
+
+You can read more about our release policy at <https://github.com/nodejs/release>.
+
+To download Node.js 22.0.0, visit: [https://nodejs.org/download/current/](/download/current/). You can find the release post at [https://nodejs.org/blog/release/v22.0.0](/blog/release/v22.0.0),
+which contains the full list of commits included in this release.
+
+### Notable Changes
+
+#### glob and globSync
+
+This release introduces `glob` and `globSync` functions for pattern matching.
+Developers can now utilize these functions for matching file paths based on specified patterns.
+
+Contributed by Moshe Atlow [#51912](https://github.com/nodejs/node/pull/51912)
+
+#### Maglev
+
+V8's Maglev Compiler is now enabled by default on supported architectures (<https://v8.dev/blog/maglev>).
+Maglev improves CPU performance for short-lived CLI programs.
+Contributed by Keyhan Vakil in [#51360](https://github.com/nodejs/node/pull/51360)
+
+#### Support require()ing synchronous ESM graphs
+
+This release adds `require()` support for synchronous ESM graphs under
+the flag `--experimental-require-module`.
+
+If `--experimental-require-module` is enabled, and the ECMAScript
+module being loaded by `require()` meets the following requirements:
+
+- Explicitly marked as an ES module with a "type": "module" field in the closest package.json or a .mjs extension.
+- Fully synchronous (contains no top-level await).
+
+`require()` will load the requested module as an ES Module, and return
+the module name space object. In this case it is similar to dynamic
+`import()` but is run synchronously and returns the name space object
+directly.
+
```

Comment:
I now realize we may want to add

```suggestion
We intend to eventually enable `require(esm)` by default, without the flag.
```

I don't think people would disagree about this goal, but anyway cc @nodejs/tsc

Reply:
That sounds implicit imho that an experimental feature might be enable by default, I'm +1 on specifying anyways